### ISSUE TITLE
Original amount improvement

### ISSUE THREAD

**Issue by mtotschnig** [mtotschnig/MyExpenses#459]
Related to, but not dependent on #458:
1) When entering the original amount, the exchange rate should be shown, same as with the equivalent amount.
2) The exchange rate could be pre-filled, same as with the exchange rate of foreign currency transfers. Until implementation of #458, the pre-filled rate could be calculated  based on the last transaction, or stored in preference (Compare with #259)
3) Optionally, user should have the option to have original amount always visible (globally, or on account level?) without having to make it appear via menu command.

**Comment by metal450** [comment]
Also: Original Amount (currency & value) are currently missing from CSV exports

**Comment by mtotschnig** [comment]
1 and 2 are now implemented. I am not sure how relevant point 3 is. Please open a new issue if you think it is.

**Comment by raphodn** [comment]
Hi!

On this subject, it would be great if we could have the option to 'remove' the original amount line.
- example 1: New transaction, I go to the menu and select 'Original Amount', and then change my mind and wish to remove it
- example 2: I clone an existing transaction that has an original amount, but wish to remove it for this new transaction

(didn't want to open a new issue because it's related to the Original amount workflow. sorry in advance)
using version 3.0.3.1 (android, f-droid)

**Comment by mtotschnig** [comment]
@raphodn Thank you for the suggestion. Will try to implement it for upcoming 3.0.5 (3.0.4 should be available on F-Droid shortly)

**Comment by mtotschnig** [comment]
@metal450 It took a long time, but will add this to 3.7.7: #1443
